governance/polls/SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024.md

```diff
@@ -0,0 +1,68 @@
+---
+title: SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024
+summary: Signal your support or opposition to onboarding PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI vault.
+discussion_link: https://forum.sky.money/t/oct-3-2024-proposed-changes-to-spark-for-upcoming-spell/25293
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-10-07T16:00:00
+end_date: 2024-10-10T16:00:00
+---
+# Poll: SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Ecosystem team. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9_Weekly_Governance_Cycle-95671ce7-60d8-4d8e-b9b6-de7493560308|0db3) will be active for three days beginning on Monday, October 7 at 16:00 UTC.
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to onboarding the two additional Pendle sUSDe PT maturities PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to the Morpho Spark DAI vault with the configurations that follow, as [the currently listed Pendle sUSDe PT](https://morpho.blockanalitica.com/ethereum/markets/8f46cd82c4c44a090c3d72bd7a84baf4e69ee50331d5deae514f86fe062b0748) is close to expiration:
```

Comment:
```suggestion
The community can vote in this poll to express support or opposition to onboarding the two additional Pendle sUSDe PT maturities PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to the Morpho Spark DAI vault with the configurations that follow:
```

Reply:
This additional context has the potential to bias voters. While not incorrect, we do not believe it is appropriate for the review section.

Reply:
We agree, thanks JS!